ReportTest.java

```diff
@@ -36,53 +33,18 @@
     public void setUp()
     {
         System.setOut(new PrintStream(outContent));
-        File directory = new File("submissions");
-        File[] sb2s = directory.listFiles();
-        Arrays.sort(sb2s);
-        Submission[] submissions = new Submission[sb2s.length];
-        for (int i = 0; i < submissions.length; i++)
-        {
-            submissions[i] = new Submission(sb2s[i]);
-            submissions[i].convertToZip();
-            submissions[i].unZip();
-            submissions[i].parseJSONFile();
-        }
         scattReport = new Report(submissions);
```

Comment:
Even though your test is not checking the content of the report yet, you still need to create the Submission objects (see removed code on left), because this gets passed into the Report constructor to build the Report object. Code does not currently compile.

I might also suggest, since you have built in the content to the Report class already, perhaps we go ahead with writing a test for the content as well? @KaraBeason @caldwellc1 we can discuss as a group on Friday to see if we think this is reasonable since the testing of the existence of the file does seem simple - but simple might not be bad for this iteration/issue.